From bddde6f3d75afb734862150e2fb80af9cf3e00f9 Mon Sep 17 00:00:00 2001
From: liyunfei <liyunfei33@huawei.com>
Date: Mon, 28 Aug 2023 11:09:03 +0800
Subject: [PATCH] turbostat clang compile support

Modified Makefile with tools/scripts/Makefile.include for clang compile and clang cross compile support.
---
 tools/power/x86/turbostat/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/power/x86/turbostat/Makefile b/tools/power/x86/turbostat/Makefile
index 92e139b..f4de368 100644
--- a/tools/power/x86/turbostat/Makefile
+++ b/tools/power/x86/turbostat/Makefile
@@ -1,5 +1,5 @@
 # SPDX-License-Identifier: GPL-2.0
-CC		= $(CROSS_COMPILE)gcc
+include ../../../scripts/Makefile.include
 BUILD_OUTPUT	:= $(CURDIR)
 PREFIX		?= /usr
 DESTDIR		?=
-- 
2.28.0.windows.1

